webhook/csv.go

```diff
@@ -82,6 +92,34 @@ func (r *ClusterServiceVersionDeploymentScaler) Handle(ctx context.Context, req
 	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledCsv)
 }
 
+func (r *ClusterServiceVersionDeploymentScaler) loadOdfConfigMapData(ctx context.Context, logger logr.Logger) error {
+
+	configmap, err := controllers.GetOdfConfigMap(ctx, r.Client, logger)
+	if err != nil {
+		return err
+	}
+
+	r.mutex.Lock()
+	defer r.mutex.Unlock()
+
+	if configmap.ResourceVersion == r.odfOperatorConfigMapResourceVersion {
+		return nil
+	}
+
+	r.odfOwnedCsvNames = map[string]bool{}
```

Comment:
I was under the impression that you like to implement sets  using `map[string]struct{}`

Reply:
It depends on the usecase.